pkg/cli/admin/release/extract.go

```diff
@@ -196,23 +210,86 @@ func (o *ExtractOptions) Run() error {
 				To:   dir,
 			},
 		}
+		var manifestErrs []error
 		found := false
 		opts.TarEntryCallback = func(hdr *tar.Header, _ extract.LayerInfo, r io.Reader) (bool, error) {
-			if hdr.Name != o.File {
+			if !o.ExtractManifests {
```

Comment:
nit: you can avoid the double-negative of `if !a {b} else {c}` by inverting the blocks to `if a {c} else {b}`.  Not a big deal, but makes for easier reading for me, anyway.

Reply:
I won't claim that one. Reused existing code and yeah, could have cleaned up, but was trying to minimize changes in the interest of making review easier (I've been dinged on that before) and generally minimizing changes.